documentation/modules/con-securing-client-authorization.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// assembly-securing-kafka-clients.adoc
+
+[id='con-securing-client-authorization-{context}']
+= User authorization
+
+User authorization is configured using the `authorization` property in `KafkaUser.spec`.
+The authorization type enabled for a user is specified using the `type` field.
+
+If no authorization is specified, the User Operator does not provision any access rights for the user.
+
+To use simple authorization, you set the `type` property to `simple` in `KafkaUser.spec`.
+Simple authorization uses the default Kafka authorization plugin, `SimpleAclAuthorizer`.
+
+Alternatively, if you are using OAuth 2.0 token based authentication, you can also xref:assembly-oauth-authorization_str[configure OAuth 2.0 authorization].
```

Comment:
You might need to edit it for better english, but I think we should have something like this:

```suggestion
Alternatively, you can use the Open Policy Agent authorization, or if you are using OAuth 2.0 token based authentication, you can also xref:assembly-oauth-authorization_str[configure OAuth 2.0 authorization].
Open Policy Agent and OAuth 2.0 authorization is not managed through `KafkaUser` resources
```